masterlist.yaml

```diff
@@ -28053,3 +28090,48 @@ plugins:
     clean:
       - crc: 0x6A181424
         util: TES5Edit v3.2.2
+  - name: 'ForgottenCity.esp'
+    dirty:
+      - <<: *dirtyPlugin
+        crc: 0x6F5FD398
+        util: 'TES5Edit v3.2.71 EXPERIMENTAL'
+        itm: 13
+    clean:
+      - crc: 0xE7C917DA
+        util: 'TES5Edit v3.2.71 EXPERIMENTAL'
+  - name: 'Whistling Mine - No Snow Under the Roof.esp'
+    clean:
+      - crc: 0x292EB6D0
+        util: 'TES5Edit v3.2.2'
+  - name: 'Helarchen Creek - No Snow Under the Roof.esp'
+    clean:
+      - crc: 0x6634ED5C
+        util: 'TES5Edit v3.2.2'
+  - name: 'Vivid Weathers.esp'
+    dirty:
+      - <<: *dirtyPlugin
+        crc: 0xC68AD9B0
+        itm: 5
+    clean:
+      - crc: 0x80AF5564
+        util: 'TES5Edit v3.2.2'
+  - name: 'Vivid Weathers - AOS patch.esp'
+    clean:
+      - crc: 0x9AF8A6B8
+        util: 'TES5Edit v3.2.2'
+  - name: 'Vivid Weathers - Extended Rain.esp'
+    dirty:
+      - <<: *dirtyPlugin
+        crc: 0xAB1CB7A0
+        itm: 10
+    clean:
+      - crc: 0x7BE42D05
+        util: 'TES5Edit v3.2.2'
+  - name: 'Vivid Weathers - Extended Snow.esp'
+    dirty:
+      - <<: *dirtyPlugin
+        crc: 0xB4B80067
+        itm: 5
+    clean:
+      - crc: 0x68C7F388
```

Comment:
A version number after CRC can be useful for future reference.
Not required but it's good to keep track of what has been checked.
      - crc: 0x68C7F388 #v0.0.0